src/libs/Firebase/index.native.ts

```diff
@@ -27,18 +23,15 @@ function startTrace(customEventName) {
                 start,
             };
         });
-}
+};
 
-/**
- * @param {String} customEventName
- */
-function stopTrace(customEventName) {
-    const stop = global.performance.now();
+const stopTrace: StopTrace = (customEventName) => {
+    // const stop = global.performance.now();
```

Comment:
If this is not needed, we should remove it

Reply:
Good point, @kubabutkiewicz Why is this commented btw?

Reply:
@kubabutkiewicz Maybe add a similar comment up here?
```suggestion
    // Uncomment to inspect logs on release builds
    // const stop = global.performance.now();
```

Reply:
I agree, let's add a line above explaining why it's commented out.